frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -1,24 +1,30 @@
 @web-terminal
 Feature: Web Terminal
-              As a user, I should be able to use web terminal
-
+              As a basic user, I should be able to use web terminal
 
         Background:
+            Given user has logged in as basic user
+              And  user is at developer perspective
```

Comment:
Not really related to your PR but can somebody tell why it matters? AFAIK terminal button live in masthead component and is available on both, admin and developer perspectives.

Reply:
That was my point from long time ago... This should be dev console interoperability, but AFAIK web terminal doesn't rely on dev console. Or am I mistaken? Is there at least some link tying dev console and web terminal together? My understanding was that web terminal was incorporated directly to web console and it's completely unrelated to dev console.

Reply:
makes sense, may be we can remove it
As part of devconsole, we created this test scenarios ;-) so it got included

Reply:
So, should i remove it?

Reply:
lets remove this step as well, thanks :-)

Reply:
Done